server/app.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const path = require("path");
+const cookieParser = require("cookie-parser");
+const logger = require("morgan");
+
+const controllers = require("./controllers");
+
+const app = express();
+
+app.use(logger("dev"));
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, "client", "build")));
+
+app.use(controllers);
+
+app.get("/", (req, res) => {
+  res.sendFile(path.join(__dirname, "..", "client", "build", "index.html"));
+});
+
+app.use((req, res, next) => {
+  res.status(404).send("Sorry - can't find that!");
```

Comment:
@thejoefriel 
I think it would be better to use `http-errors` instead of writing message manual, it will do that for you 
all you need is to pass the err status for it and it will create err object, then you can send with response

Reply:
You don't need the 404 route in Express, as all unhandled routes will be passed to React

Reply:
@RamyAlshurafa I agree that's a better approach - and if we move all API routes to be under `/api/` we can give a 404 response for any that are unhandled. But we can deal with this later :)

Reply:
As this won't break anything in the meantime, I've set up an issue #48 on this so someone can simply remove at some point. 

Happy to use http-errors on errors going forward though 👍